tests/statistics_test.sh

```diff
@@ -194,11 +196,39 @@ function month_statistics_Test
 function year_statistics_Test
 {
   local target_year="2019"
+  local year_data
   local msg="Currently, kw does not have any data for the requested year."
+  local line
 
-  ID=1
+  declare -a expected_cmd=(
+    "2020 summary"
+    "Total   Max  Min       Average"
```

Comment:
I think this will be a problem soon because this output layout will change soon, as you pointed here https://github.com/kworkflow/kworkflow/issues/267.

Maybe you can write this test in a way that we ignore these two first lines?